src/status_im/ui/screens/wallet/send/views.cljs

```diff
@@ -23,8 +23,7 @@
 (defn toolbar-view [signing?]
   [toolbar/toolbar2 {:style wallet.styles/toolbar}
    [toolbar/nav-button (act/back-white (if signing?
-                                         #(do (re-frame/dispatch [:wallet/discard-transaction])
-                                              (act/default-handler))
+                                         #(re-frame/dispatch [:wallet/discard-transaction-leave])
```

Comment:
why did you remove `(act/default-handler)` i think it's expected to close screen and go back to wallet